contracts/policies/UpgradeEventCompact.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^0.4.11;
+
+import '../controller/Controller.sol';
+import '../ownership/Ownable.sol';
+
+contract UpgradeEventCompact {
+
+  // states
+  //  - verifying, initial state
+  //  - controlling, after verifying, before complete
+  //  - complete, after controlling
+  enum EventState { Verifying, Complete }
+  EventState public state;
+
+  // Terms
+  address public nextController;
+  address public oldController;
+  address public council;
+
+  // Params
+  address powerEventReplacement;
+  address nextPullPayment;
+  address storageAddr;
+  address nutzAddr;
+  address powerAddr;
+  uint256 maxPower;
+  uint256 downtime;
+  uint256 purchasePrice;
+  uint256 salePrice;
+
+  function UpgradeEventCompact(address _oldController, address _nextController, address _nextPullPayment) {
+    state = EventState.Verifying;
+    nextController = _nextController;
+    oldController = _oldController;
+    nextPullPayment = _nextPullPayment; //the ownership of this satellite should be with oldController
+    council = msg.sender;
+  }
+
+  modifier isState(EventState _state) {
+    require(state == _state);
+    _;
+  }
+
+  function tick() public {
```

Comment:
looks like `tick` has the same checks as `verifyComplete`. Probably we don't need tick at all, can just call `verifyComplete` directly. Or better rename it to `upgrade` ?

Reply:
Yeah. Sounds good. Will do.